source/messaging/sharing-files.rst

```diff
@@ -54,4 +54,4 @@ Attachment Limits and Sizes
 
 Up to five files can be attached per post. The default maximum file size is 100 MB, but this can be changed by the System Admin.
 
-Image files can be a maximum size of 6048 pixels x 4032 pixels, or 24 MP (mega pixels), or a raw image file size of approximately 36 MB.
+Image files can be a maximum size of 6048 pixels x 4032 pixels, with a maximum image resolution of 33 MP (mega pixels) or 8K resolution, and a maximum raw image file size of approximately 36 MB. System Admins can customize the maximum image resolution size within the ``config.json`` file. See our `Configuration Settings <https://docs.mattermost.com/configure/configuration-settings.html#maximum-image-resolution>`__ product documentation for details.
```

Comment:
Maximum resolution default is 7680x4320 pixels now (it can be extended by modifying the provided config setting). The raw image size can be up to ~253MB with the default.